### DISCUSSION TITLE
new page not linked?

### DISCUSSION THREAD

**Discussion by Makuna** [Makuna/NeoPixelBus#398] [Wiki]
@sandandstars, "A method for resolving this" page was created by you, but I didn't see a change elsewhere that linked to this.

See the new guidelines at to the top of this Category about page content guidelines.

**Comment by sandandstars** [comment]
Sorry, I've not added to Github before.
I thought the suggestion would be helpful and it seemed like a good place for it; I would delete it now but I can't find it.
S&S

**Comment by Makuna** [comment]
I like the blurb, just need to make it fit in and discoverable.  

BUT, now with this new discussion area, you can just post it here as a suggestion for addition ;-)
